system_test/aest_hard_fork_SUITE.erl

```diff
@@ -619,19 +623,20 @@ new_node_can_mine_spend_tx_on_old_chain_using_new_protocol(Cfg) ->
     ReceivedBalance = get_balance(new_node3, PubKey1, Cfg),
     ct:log("Balance of node ~p with public key ~p is ~p", [new_node1, PubKey1, ReceivedBalance]),
     ?assertEqual(BalanceToSpend, ReceivedBalance),
+    %% new_node1 has one spend transaction.
+    [#{block_hash := SpendTxBlockHash, block_height := SpendTxHeight, tx := TxInfo}] =
+        get_account_txs(new_node3, PubKey1, Cfg),
+    ?assertEqual(ReceivedBalance, maps:get(amount, TxInfo)),
+    ?assertEqual(Fee, maps:get(fee, TxInfo)),
+    ?assertEqual(PubKey3, maps:get(sender, TxInfo)),
+    ?assertEqual(PubKey1, maps:get(recipient, TxInfo)),
+    %% Block with spend transaction has new protocol version.
+    ?assertEqual(10, maps:get(version, get_block_by_hash(new_node3, SpendTxBlockHash, Cfg))),
```

Comment:
I see test had already checked the protocol version before posting spend tx (`?assertEqual(10, maps:get(version, B3NewProtocol))`) hence in addition to this line reasonably guarantees that spend tx was mined in block of version 10. Thanks.